### DISCUSSION TITLE
Is there any way to communicate through Android serial port?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#3323] [Q&A]
As far as I know, AIR doesn't seem to have a method for Android serial communication, does it?
I would like to ask if there are any ANEs that can implement this function?
thank you!

**Comment by marchbold** [comment]
We did a small one for a client many years ago but don't have anything that we released. I'll look into what's involved. Do you have any requirements?

**Reply by Jason0247** [reply]
I want to use the playback box of an Android host for serial communication and control some devices.
